### PR TITLE
Support prebuilds

### LINKED ISSUE #3: Support prebuilds

### PR DESCRIPTION
**Pull request by staltz**
Closes #3

### CODE REVIEW

**Comment by staltz** [comment]
@mycognosist CI should have released to npm but it didn't, I'm not sure why, but I suspect that it *would* run in the `main` branch if it were merged.

File: README.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ npm run perf
 
 The default values for the performance benchmarks (`test/perf.js`) are 100 messages from 1 author, for a total of 10 iterations. These value constants can be changed in `test/perf.js`. Performance benchmarks for the multi-author method default to 100 messages from 5 authors, for a total of 10 iterations (`test/multiAuthorPerf.js`).
 
+## Releasing new versions
+
+To release a new version, all that you need to do is update the version number in `package.json` and commit with a message that starts with the word "release", e.g. `release 1.1.0`. Then, CI (GitHub Actions) will detect that, and compile this library for many variations of operating system and Node.js versions and then publish those as prebuilds to NPM. This repository has an environment variable `NPM_TOKEN` set up so that GitHub Actions has publish permissions for this package.
```
Comment (mycognosist) [inline README.md:47]: 👍🏻 👍🏻  Great to have this documented so clearly.

**Review by mycognosist — APPROVED** [review]
Rad rad 🌱

**Comment by staltz** [comment]
I'll merge but I may have to tweak the CI on the `main` branch if things don't work. I'll try to not mess up :D

**Pull request merged**